styles.css

```diff
@@ -0,0 +1,285 @@
+
+.header > .list {
+  display: flex;
+  list-style-type: none;
+  justify-content: space-between;
+  font-weight: bold;
+}
+
+.firstsection {
```

Comment:
Your first section is nicely laid out. I like the rounded borders to make it seem more like a button; Without it, a user might not roll over it to get to the link.  Very nice!